src/components/contributors.tsx

```diff
@@ -15,6 +15,10 @@ const contributors: GitHubProfileProps[] = [
         userName: 'Kumbong',
         role: 'Tech Lead',
     },
+    {
+        userName: 'EJAyarma',
+        role: 'Developer',
+    }
```

Comment:
Good! Can you please specify the actual role assigned to you. You can find it in the scoping documents on Confluence

Reply:
Sure.